src/__tests__/http-test.ts

```diff
@@ -1572,7 +1573,7 @@ function runTests(server: Server) {
       const response = await app.request().put(urlString({ query: '{test}' }));
 
       expect(response.status).to.equal(405);
-      expect(response.headers.allow).to.equal('GET, POST');
+      expect(response.get('allow')).to.equal('GET, POST');
```

Comment:
`headers` was a private property

Reply:
This is also not a type change and can be done before TS convertion